### PR TITLE
Initial commit of MetaNovo tool wrapper.

### PR DESCRIPTION
**Pull request by reid-wagner**
Adds MetaNovo tool wrapper.

From the [project](https://github.com/uct-cbio/proteomics-pipelines/) README:

> MetaNovo is an open-source pipeline that integrates existing tools with a custom algorithm to produce targeted databases for mass spectrometry analysis. As inputs it requires raw mass spectrometry data in MGF format, and a UniProt FASTA file to search.

### CODE REVIEW

File: tools/metanovo/metanovo.xml [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+  <macros>
+    <token name="@TOOL_VERSION@">1.9.4</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <import>macros_modifications.xml</import>
+  </macros>
+  <command>
+    <![CDATA[
+      #set $mgf_dir = 'mgf_files'
+      mkdir $mgf_dir &&
+      ln -s $input_fasta in.fasta &&
```
Comment (jj-umn) [inline tools/metanovo/metanovo.xml:17]: single quotes around dataset names

Comment (jj-umn) [reply tools/metanovo/metanovo.xml:17]: ln -s '$input_fasta' in.fasta

File: tools/metanovo/metanovo.xml [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+
+      #if $input_type.type == "collection"
+      #for $input in $input_type.input_mgf_collection:
+      ln -s $input $mgf_dir/$input.element_identifier &&
```
Comment (jj-umn) [inline tools/metanovo/metanovo.xml:21]: single quotes: ln -s '$input' '$mgf_dir/$input.element_identifier'

**Review by jj-umn — CHANGES_REQUESTED** [review]

File: tools/metanovo/metanovo.xml (hunk 2 above)
Comment (reid-wagner) [reply tools/metanovo/metanovo.xml]: Thanks @jj-umn. I've added single quotes for user input, and problem characters in the filenames are now replaced.

**Pull request merged**